apps/src/componentLibrary/breadcrumbs/Breadcrumbs.tsx

```diff
@@ -1,120 +1,70 @@
 import classNames from 'classnames';
-import uniq from 'lodash/uniq';
 import React from 'react';
 
 import {ComponentSizeXSToL} from '@cdo/apps/componentLibrary/common/types';
-
-import BreadCrumb from './_Breadcrumb';
+import FontAwesomeV6Icon from '@cdo/apps/componentLibrary/fontAwesomeV6Icon';
+import Link, {LinkBaseProps} from '@cdo/apps/componentLibrary/link';
 
 import moduleStyles from './breadcrumbs.module.scss';
 
+interface BreadcrumbProps extends LinkBaseProps {
+  text: string;
+  href: string;
```

Comment:
`href` is already defined on `LinkBaseProps` right?

Reply:
Also if `text` is required, should just extend/be `LinkWithText`?

Reply:
Yes, thanks for noticing!
I added this initially for better documentation, but it looks like it makes no significant difference to create a separate type here, will allow `LinkWithText` type import in a separate PR and will use it here

Reply:
https://github.com/code-dot-org/code-dot-org/pull/61281